exercises.md

```diff
@@ -1,7 +1,10 @@
 # Exercises
 
+<<<<<<< HEAD
+=======
```

Comment:
Looks like this still has a couple artifacts from a rebase

Reply:
Thanks for the heads up - I forgot to pull the most recent commit to my local branch.